Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.7-alpine as build
+WORKDIR /build
+RUN pip install virtualenv
+RUN virtualenv /build
+RUN apk add --update --no-cache \
+    python \
+    python-dev \
+    build-base \
+    linux-headers \
+    pcre-dev \
+    py-pip \
+    curl \
+    openssl \
+    git
+COPY requirements.txt /build
+RUN /build/bin/pip install -r requirements.txt uwsgi
+
+FROM python:3.7-alpine as application
+RUN adduser -D -g '' uwsgi
+RUN apk add --no-cache git
+WORKDIR /app
+COPY --from=build /build /build
+COPY --from=build /usr/lib /usr/lib
+COPY . /app/
```

Comment:
we should be able to install gman with pip instead of copying locally actually

Reply:
Sure we can make that change